src/ui/styles/_buttons.scss

```diff
@@ -9,7 +9,6 @@
 .ch-btn:focus {
     line-height: $btn-line-height;
     padding: $btn-padding;
-    margin: $btn-margin;
```

Comment:
@jota-v Be careful by removing margin entirely and not only for :focus state. The only separator in this case is a space that can be removed by some HTML minifier or just by markup concept, e.g.
![No space](https://www.evernote.com/shard/s4/sh/ad4499ed-3f35-4434-9a05-ead8ed500877/6e776d55b3eb8226/res/85b3053c-4453-4ea8-aff6-14bf02e6e05e/skitch.png)